### ISSUE TITLE
Uncaught Error: Element type is invalid: expected a string

### ISSUE THREAD

**Issue by ablbol** [reactstrap/reactstrap#493]
I just upgraded to reactstrap version 4.8.0 and started seeing the following warnings and errors while launching modal dialogue.  I went down to version 4.6.2 since the error did not show up there. 

invariant.js:44 Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `Alert`.
    at invariant (invariant.js:44)
    at ReactCompositeComponentWrapper.instantiateReactComponent [as _instantiateReactComponent] (instantiateReactComponent.js:74)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:366)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:257)
    at Object.mountComponent (ReactReconciler.js:45)
    at ReactDOMComponent.mountChildren (ReactMultiChild.js:236)
    at ReactDOMComponent._createInitialChildren (ReactDOMComponent.js:703)
    at ReactDOMComponent.mountComponent (ReactDOMComponent.js:522)
    at Object.mountComponent (ReactReconciler.js:45)
    at ReactDOMComponent.mountChildren (ReactMultiChild.js:236)

**Comment by TheSharpieOne** [comment]
The issue is mostly like that you do not have a compatible version of [react-transition-group](https://github.com/reactjs/react-transition-group) installed; which is a peerDep. The latest (what is installed with `npm i react-transition-group`) is 2.x, but reactstrap [requires `^1.1.2`](https://github.com/reactstrap/reactstrap/blob/master/package.json#L99)
I have tried it with react-transition-group v2.0.2 and saw what you saw, but only changing react-transition-group down to v1.2.0 fixes the issue.
Probably need to see what the breaking change is, and hopefully support both versions. If reactstrap cannot support both version, then it would probably be best to make a change to drop support for react-transition-group v1 and move to react-transition-group v2.

***tl;dr** use react-transition-group v1.2.0; react-transition-group v2 does not appear to work.*

**Comment by ablbol** [comment]
Thanks
